pydis_site/apps/content/resources/guides/pydis-guides/contributing/bot.md

```diff
@@ -2,207 +2,643 @@
+    # before message making a channel dormant
+    deleted_idle_minutes: 1
+
+    # Maximum number of channels to put in the available category
+    max_available: 2
+
+    # Maximum number of channels across all 3 categories
+    # Note Discord has a hard limit of 50 channels per category, so this shouldn't be > 50
+    max_total_channels: 20
+
+    # Prefix for help channel names
+    name_prefix: 'help-'
+
+    # Notify if more available channels are needed but there are no more dormant ones
+    notify: true
+
+    # Channel in which to send notifications
+    notify_channel: *DEV_LOUNGE_HELPERS_ORG
+
+    # Minimum interval between helper notifications
+    notify_minutes: 5
+
+    # Mention these roles in notifications
+    notify_roles:
+        - *HELPERS_ROLE
+
+##### <<  Add any additional sections you need to override from config-default.yml  >> #####
+            </code>
+          </pre>
+</div></div></div>
+<br>
 
----
-# Configure the bot
-You will need to copy IDs of the test Discord server, as well as the created channels and roles to paste in the config file.
-If you're not sure how to do this, [check out the information over here.](../setting-test-server-and-bot-account#obtain-the-ids)
-
-1. Create a copy of `config-default.yml` named `config.yml` in the same directory.
-2. Set `guild.id` to your test servers's ID.
-3. Change the IDs in the [sections](#server-setup) mentioned earlier to match the ones in your test server.
-4. Set `urls.site_schema` and `urls.site_api_schema` to `"http://"`.
-5. Set `urls.site`:
-    - If running the webserver in Docker, set it to `"web:8000"`.
-        - If the site container is running separately (i.e. started from a clone of the site repository), then [COMPOSE_PROJECT_NAME](../docker/#compose-project-names) has to be set to use this domain. If you choose not to set it, the domain in the following step can be used instead.
-    - If running the webserver locally and the hosts file has been configured, set it to `"pythondiscord.local:8000"`.
-    - Otherwise, use whatever domain corresponds to the server where the site is being hosted.
-6. Set `urls.site_api` to whatever value you assigned to `urls.site` with `api` prefixed to it, for example if you set `urls.site` to `web:8000` then set `urls.site_api` to `api.web:8000`.
-7. Setup the environment variables listed in the section below.
-
-### Environment variables
-
-These contain various settings used by the bot.
-To learn how to set environment variables, read [this page](../configure-environment-variables) first.
+If you don't wish to use the provided `config.yml` above, you can copy the `config-default.yml` file, rename it, and change whatever values you need. Note that you don't need to specify all items in `config.yml`, just the ones you want to override such as channel ID's. 
 
-The following is a list of all available environment variables used by the bot:
+See [here](../obtaining-discord-ids) for help with obtaining Discord ID's.
```

Comment:
Can we make this standout with some sort of callout? It's currently getting lost in between all the text and I think it would be helpful to mark this as a "this is useful info! Check it out if you need it".

Something like this: 
![image](https://user-images.githubusercontent.com/20641196/134241302-b6c846eb-e015-4cfe-b131-4ac9f484a5f8.png)
(ref: https://pythondiscord.com/pages/guides/pydis-guides/contributing/sir-lancebot/)

Reply:
I highlighted it, but it's still inside the collapsible. You think it should be outside?